docs/requirements.md

```diff
@@ -1,51 +1,143 @@
+  - There is no concept for primary regions in DynamoDB, so you must designate a primary region by yourself.
 
-ScalarDB does not work on some Cassandra-compatible databases that do not support LWTs, such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/). This is because the Consensus Commit transaction manager relies on the linearizable operations of underlying databases to make transactions serializable.
+#### Analytics
+{:.no_toc}
+- Not applicable. DynamoDB always returns committed records, so there are no DynamoDB-specific requirements.
 
-## CosmosDB database requirements
+</div>
 
-In your Azure CosmosDB account, you must set the **default consistency level** to **Strong**.
+<div id="Cosmos_DB_for_NoSQL" class="tabcontent" markdown="1">
 
-Consensus Commit, the ScalarDB transaction protocol, requires linearizable reads. By setting the **default consistency level** to **Strong**, CosmosDB can guarantee linearizability.
+#### Transactions
+{:.no_toc}
+- Use a single primary region for all operations with `Strong` or `Bounded Staleness` consistency.
 
-For instructions on how to configure this setting, see the official documentation at [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level).
+#### Analytics
+{:.no_toc}
+- Not applicable. Cosmos DB always returns committed records, so there are no Cosmos DB–specific requirements.
 
-## JDBC database recommendations
+</div>
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+<div id="Cassandra" class="tabcontent" markdown="1">
 
-For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+#### Transactions
+{:.no_toc}
+- Use a single primary cluster for all operations (no read or write operations in non-primary clusters).
+- Use `batch` or `group` for `commitlog_sync`.
+- If you're using Cassandra-compatible databases, those databases must properly support lightweight transactions (LWT).
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+#### Analytics
+{:.no_toc}
+- Not applicable. Cassandra always returns committed records, so there are no Cassandra-specific requirements.
+
+</div>
+</div>
```

Comment:
I just realized that the tabs should match the same order as we have in other docs (like [when setting up your database for ScalarDB](https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb/#set-up-your-database-for-scalardb)). Can we change the order of the tabs as follows?:

- Cassandra
- Cosmos DB for NoSQL
- DynamoDB
- JDBC databases

```suggestion
<div id="tabset-1">
<div class="tab">
  <button class="tablinks" onclick="openTab(event, 'Cassandra', 'tabset-1')" id="defaultOpen-1">Cassandra</button>
  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL', 'tabset-1')">Cosmos DB for NoSQL</button>
  <button class="tablinks" onclick="openTab(event, 'DynamoDB', 'tabset-1')">DynamoDB</button>
  <button class="tablinks" onclick="openTab(event, 'JDBC_databases', 'tabset-1')">JDBC databases</button>
</div>

<div id="Cassandra" class="tabcontent" markdown="1">

#### Transactions
{:.no_toc}
- Use a single primary cluster for all operations (no read or write operations in non-primary clusters).
- Use `batch` or `group` for `commitlog_sync`.
- If you're using Cassandra-compatible databases, those databases must properly support lightweight transactions (LWT).

#### Analytics
{:.no_toc}
- Not applicable. Cassandra always returns committed records, so there are no Cassandra-specific requirements.
</div>
<div id="DynamoDB" class="tabcontent" markdown="1">

#### Transactions
{:.no_toc}
- Use a single primary region for all operations. (No read and write operations on global tables in non-primary regions.)
  - There is no concept for primary regions in DynamoDB, so you must designate a primary region by yourself.

#### Analytics
{:.no_toc}
- Not applicable. DynamoDB always returns committed records, so there are no DynamoDB-specific requirements.
</div>

<div id="Cosmos_DB_for_NoSQL" class="tabcontent" markdown="1">

#### Transactions
{:.no_toc}
- Use a single primary region for all operations with `Strong` or `Bounded Staleness` consistency.

#### Analytics
{:.no_toc}
- Not applicable. Cosmos DB always returns committed records, so there are no Cosmos DB–specific requirements.
</div>

<div id="JDBC_databases" class="tabcontent" markdown="1">

#### Transactions
{:.no_toc}
- Use a single primary server or synchronized multi-primary servers for all operations (no read operations on read replicas that are asynchronously replicated from a primary database).
- Use read-committed or stricter isolation levels.

#### Analytics
{:.no_toc}
- Use read-committed or stricter isolation levels.

</div>
</div>
```